build_projects/dotnet-cli-build/TestTargets.cs

```diff
@@ -30,7 +30,7 @@ public class TestTargets : Task
             // "dotnet-compile-fsc.Tests",
             "dotnet-new.Tests",
             "dotnet-pack.Tests",
-            "dotnet-projectmodel-server.Tests",
+            //"dotnet-projectmodel-server.Tests",
```

Comment:
Let's get an issue tracking this and put in a comment.  Let's make that the "default" behavior when we introduce a hack.

---
In reply to: [68990220](https://github.com/dotnet/cli/pull/3696#discussion_r68990220) [](ancestors = 68990220)